packages/dds/tree/src/simple-tree/toMapTree.ts

```diff
@@ -404,10 +395,11 @@ function objectToMapTree(data: InsertableContent, schema: TreeNodeSchema): Exclu
 
 	const fields = new Map<FieldKey, ExclusiveMapTree[]>();
 
-	// Loop through field keys without data, and assign value from its default provider.
```

Comment:
This comment is explicitly incorrect in a misleading way. It is a very significant detail this does not apply defaults, and comment says the opposite. Fixed!